content/en/Platform Deep Dive/Pentests/Pentest Process/_index.md

```diff
@@ -3,10 +3,21 @@ title: "Pentest Process"
 linkTitle: "Pentest Process"
 weight: 20
 description: >
-  Learn how Cobalt tests your assets.
+  Learn about the pentest lifecycle.
 ---
 
 {{% pageinfo %}}
-Cobalt defines a sequence of events which use specific methodologies based on your asset.
+Explore the stages of the Cobalt Pentest as a Service (PtaaS) program.
 {{% /pageinfo %}}
 
+![Lifecycle of the Cobalt pentest program](/deepdive/PentestProcessFlow.png "Lifecycle of the Cobalt pentest program")
+<br><br>
+
+| | Stage | Description |
+|---|---|---|
+| 1 | **Discover** | Prepare for the pentest engagement.<ul><li>Map the attack surface of your software.</li><li>Create an account on the Cobalt platform following our invitation.</li></li>  |
+| 2 | **Plan** | Plan, scope, and schedule your pentest. To understand your expectations, we'll arrange a call with you.<ul><li>See our [Getting Started](/getting-started/) guide to learn how to launch a pentest. If you need help, [contact us](/getting-started/#get-in-touch).</li><li>Prepare the environment for our pentesters, such as set up test credentials for them.</li><li>Alert the stakeholders in your organization about the upcoming pentest.</li><li>Once you've submitted the pentest, we'll assign pentesters based on your technology stack.</li></ul> |
```

Comment:
"To understand your expectations, we'll arrange a call with you."
We should get feedback from CS about this as they want to move away from calls as much as possible.

Reply:
check with Trang, Derek, Nathan and Andrew.

Reply:
@yogipetkar I removed the part about arranging a call after confirming this with CSMs.